common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -76,7 +76,7 @@ class ComplexEncoder(json.JSONEncoder):
     """
     Extend the JSON encoder to correctly handle complex numbers
     """
-    def default(self, obj):
+    def default(self, obj):  # pylint: disable=method-hidden
```

Comment:
It's caused by the design of the json library.  `json.encode` lets you pass a `default()` function that overrides the builtin `default` method.